waku/v2/protocol/waku_filter/waku_filter.nim

```diff
@@ -182,11 +189,9 @@ method init*(wf: WakuFilter) =
   wf.codec = WakuFilterCodec
 
 proc init*(T: type WakuFilter, peerManager: PeerManager, rng: ref BrHmacDrbgContext, handler: MessagePushHandler): T =
-  new result
-  result.rng = crypto.newRng()
-  result.peerManager = peerManager
-  result.pushHandler = handler
-  result.init()
+  var wf = WakuFilter(rng: crypto.newRng(), peerManager: peerManager, pushHandler: handler)

```

Comment:
Nitpick: stylistic thing but personally prefer to have inits like `crypto.newRng()` in a temp variable. Sometimes makes a difference when it comes to debugging. Also makes line <80ch

Reply:
Makes sense! 👍